mpt/impl/smtp/cassandra-rabbitmq-swift/src/test/resources/logback.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Rename it to `logback-test.xml`

Reply:
I removed it, it's not loaded by logback anyway because of test-jar mess